.github/workflows/release.yml

```diff
@@ -21,33 +21,46 @@ jobs:
 
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           repository: wingtk/gvsbuild
           fetch-depth: 0
 
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
         with:
           python-version: ${{ matrix.python }}
           architecture: ${{ matrix.arch }}
 
       - name: Install Build Deps
-        run: python -m pip install wheel
+        run: python -m pip install wheel build
```

Comment:
Do we need this step if using poetry?

Reply:
I have not tested that mostly just updated with the new build instructions weather things can be removed now I'm not sure.

Reply:
I don't see where poetry installs wheel so I would assume yes.

Reply:
Ok I see but why did you add `build`?

Reply:
an error on my part build is not included in the new PR. Also sorry its seperate.